tests/wrapper/test_request.py

```diff
@@ -120,17 +130,66 @@ def test_request_timeout(self):
         ):
             self.headers_patcher.stop()
 
-    def test_retry_request(self):
+    @MockedRequestsTestCase.no_auth
+    def test_request_timeout_2(self):
         with self.given(
             """
             - uma requisição qualquer
             """
         ):
+            url = "http://foo.com"
+
             self.headers_patcher = patch(
                 "bb_wrapper.wrapper.request.RequestsWrapper._get_request_info"
             )
             self.mocked_headers = self.headers_patcher.start()
 
+            def raise_timeout(request):
+                return self.raise_timeout()
+
+            self.mocked_headers.return_value = {}
+            self.mock_responses.add_callback(
+                responses.GET,
+                url,
+                callback=raise_timeout,
+            )
+            wrapper = RequestsWrapper(base_url="", timeout=2)
+
+        with self.when(
+            """
+            - o servidor demorar X segundos para responder
```

Comment:
Acho que fica mais claro assim, eu fiquei me perguntando qual era o valor de X rsrs

```suggestion
            - o servidor demorar tempo o suficiente para ocorrer timeout
```